servicemesh-install/topics/appendix-threescale-adapter.adoc

```diff
@@ -69,8 +94,32 @@ The adapter embeds a tool that allows generation of the `handler`, `instance`, a
 |File to save produced manifests to
 |No
 |Standard output
+
+|`-v`
+|Outputs the CLI version and exits
+|No
+|
 |===
 
+=== Generate templates from URL examples
+This example generates the templates from the URL, service, and personal token:
+
+----
+3scale-gen-config --url="https://<organization>-admin.3scale.net:443" --service="123456789" --token="[redacted]"
```

Comment:
Prefix commands that don't need root access with $ and ones that do need root access with #

Reply:
Fixed.